test/e2e/usersignup_test.go

```diff
@@ -109,6 +109,63 @@ func (s *userSignupIntegrationTest) TestAutomaticApproval() {
 	})
 }
 
+func (s *userSignupIntegrationTest) TestProvisionToOtherClusterWhenOneIsFull() {
+	s.T().Run("set per member clusters max number of users for both members and expect that users will be provisioned to the other member when one is full", func(t *testing.T) {
+		// given
+		var memberLimits []test.PerMemberClusterOption
+		toolchainStatus, err := s.hostAwait.WaitForToolchainStatus(wait.UntilToolchainStatusHasConditions(ToolchainStatusReadyAndUnreadyNotificationNotCreated()...))
+		require.NoError(t, err)
+		for _, m := range toolchainStatus.Status.Members {
+			if s.memberAwait.ClusterName == m.ClusterName {
+				memberLimits = append(memberLimits, test.PerMemberCluster(s.memberAwait.ClusterName, m.UserAccountCount+1))
+			} else if s.memberAwait2.ClusterName == m.ClusterName {
+				memberLimits = append(memberLimits, test.PerMemberCluster(s.memberAwait2.ClusterName, m.UserAccountCount+1))
+			}
+		}
+		require.Len(s.T(), memberLimits, 2)
+
+		userAccounts := &v1alpha1.UserAccountList{}
+		err = s.memberAwait.Client.List(context.TODO(), userAccounts)
+		require.NoError(s.T(), err)
+
+		userAccounts = &v1alpha1.UserAccountList{}
+		err = s.memberAwait2.Client.List(context.TODO(), userAccounts)
+		require.NoError(s.T(), err)
+
+		s.hostAwait.UpdateHostOperatorConfig(test.AutomaticApproval().Enabled().MaxUsersNumber(0, memberLimits...))
+
+		// when
+		userSignup1, mur1 := s.createAndCheckUserSignup(false, "multimember-1", "multi1@redhat.com", nil, ApprovedAutomatically()...)
+		// we need to sleep one second to create UserSignup with different creation time
+		time.Sleep(time.Second)
+		userSignup2, mur2 := s.createAndCheckUserSignup(false, "multimember-2", "multi2@redhat.com", nil, ApprovedAutomatically()...)
+
+		// then
+		expectedCluster1 := s.memberAwait
+		expectedCluster2 := s.memberAwait2
+		if mur1.Spec.UserAccounts[0].TargetCluster == s.memberAwait2.ClusterName {
+			expectedCluster1 = s.memberAwait2
+			expectedCluster2 = s.memberAwait
+		}
+		t.Logf("TargetCluster1: %s", mur1.Spec.UserAccounts[0].TargetCluster)
+		t.Logf("TargetCluster2: %s", mur2.Spec.UserAccounts[0].TargetCluster)
+		require.Equal(s.T(), mur1.Spec.UserAccounts[0].TargetCluster, expectedCluster1.ClusterName)
+		require.Equal(s.T(), mur2.Spec.UserAccounts[0].TargetCluster, expectedCluster2.ClusterName)
```

Comment:
wouldn't be easier just to verify that they are provisioned to different clusters?
```suggestion
		require.NotEqual(s.T(), mur1.Spec.UserAccounts[0].TargetCluster,mur2.Spec.UserAccounts[0].TargetCluster)
```

BTW, when using `Equal` - the first param is the expected one:
```
require.Equal(s.T(), expectedCluster1.ClusterName, mur1.Spec.UserAccounts[0].TargetCluster)
```

Reply:
Good point, thanks!